Nos/Assets/Assets.xcassets/tab-profile.imageset/Contents.json

```diff
@@ -52,5 +52,8 @@
   "info" : {
     "author" : "xcode",
     "version" : 1
+  },
+  "properties" : {
+    "template-rendering-intent" : "original"
```

Comment:
@mplorentz this fixes the issue with the profile icon. I'm surprised we didn't see the issue on previous builds but I guess maybe something changed in Xcode 16 or 16.1.